canonical-web-and-design/vanilla-framework
# Use code snippet and iframe for examples instead of CodePen embeds

## Done

- Replaces CodePen embeds with code snippet and custom iframe
- Adds Prism to allow syntax highlighting of the examples
- Resizes the embedded iframes to the (estimated) height of the example

Fixes #3556 #3557

## QA

- Open [demo](https://vanilla-framework-3570.demos.haus/docs)
- Browse the docs and check embedded examples
- Make sure to verify various different pages
- Find examples with JS, make sure you can change the code snippet code via dropdown
- Find examples of different heights and verify they render at expected height
- Check in both Chrome and Firefox to make sure embed code works

## Screenshots

<img width="946" alt="Screenshot 2021-02-19 at 13 56 54" src="https://user-images.githubusercontent.com/83575/108507721-05ff4b80-72bb-11eb-8452-08d831f03b2d.png">
<img width="942" alt="Screenshot 2021-02-19 at 13 58 24" src="https://user-images.githubusercontent.com/83575/108507724-07c90f00-72bb-11eb-82ab-30cbc1170918.png">
<img width="942" alt="Screenshot 2021-02-19 at 13 58 07" src="https://user-images.githubusercontent.com/83575/108507725-08fa3c00-72bb-11eb-9053-87b8e11dc27c.png">
